.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
+
+Fixes #
```

Comment:
We should put directions for users in a markdown comment, and put only the template of the PR we want in markdown.

```markdown
<!-- 

Please review guidelines

-->

- [ ] Have you reviewed the guidelines?
 
```

 Thoughts?

Reply:
I am not sure I would want a checklist for users to do in the templates.  Do you think that is a good idea?  Also I am not sure the link will work if we put that in a comment?

Reply:
The link won't work, but I think it's fine to provide the high level instructions there, e.g. following.

```
<!-- 
Please make sure you fill the following sections. If this PR fixes an issue, please tag the issue number in the first section.

e.g.

This fixes issue #123
-->

- [ ] I have reviewed the [Contributing Guidelines](../CONTRIBUTING.md)
- [ ] I agree to license my contributions under the same license as in this repository
- [ ] I have verified that the tests pass

# Description

<!-- Describe your Pull Request here -->

# Proposed changes

<!-- Describe the proposed changes of Pull Request here -->

-
- 
-

```

The checklist is useful for list of things people should consider before submitting the PR. What do you think of the above example?

Reply:
I am OK with that,  if we put the link in comments it should be the global link, not a relative one though.  @kdheepak Can you add a commit with the changes?,  then we can merge this.

Reply:
I can make the change later tonight.